compiler/src/utils/wasm_utils.re

```diff
@@ -279,6 +286,27 @@ let get_wasm_sections = (~reset=false, inchan) => {
             );
           let true_offset = pos_in(inchan);
           (Custom(name), true_offset, size - (true_offset - offset));
+        | Export(_) =>
+          let num_exports = Int32.to_int(read_leb128_u32_input(inchan));
+          let rec read_export = () => {
+            let name_len = Int32.to_int(read_leb128_u32_input(inchan));
+            let name = really_input_string(inchan, name_len);
```

Comment:
gets me every time

Reply:
but _really_

Reply:
I 'member when y'all were writing this function yourselves throughout the compiler 😉